package.json

```diff
@@ -68,9 +68,10 @@
     "typescript": "^2.4.0"
   },
   "scripts": {
-    "clean:dist": "rimraf dist/*",
-    "clean:docs": "rimraf docs/*",
+    "clean:storybook": "rimraf build/storybook/*",
     "clean:build": "rimraf build/*",
+    "clean:dist": "rimraf dist/*",
+    "clean": "run-p clean:*",
```

Comment:
since storybook is now in the build dir this could be
`"clean": "run-p clean:build clean:dist`